### PR TITLE
Update firefox checkver

### PR DESCRIPTION
**Pull request by linsui**
Fix https://github.com/lukesampson/scoop-extras/issues/2340 ?

### CODE REVIEW

**Comment by linsui** [comment]
No matter standard edition or esr and beta edition will creat `Firefox` shortcut in installtion. For esr, should I change the shortcut to `Firefox Esr` as beta use `Firefox Beta` ?

**Comment by Ash258** [comment]
https://github.com/lukesampson/scoop-extras/blob/a6ceb22444d0e6ee3adbffa305d40c58ad9d713e/bucket/firefox-developer.json#L25-L31

**Comment by linsui** [comment]
Firefox Developer Edition will creat a shortcut `Firefox Developer Edition` in installation, but esr and beta will creat a shortcut `Firefox`. So I wonder we should keep the `Firefox` or make it more clear.

**Comment by Ash258** [comment]
I hoped it is clear... You should never replace binaries and shortcuts. Rename them.

File: bucket/firefox-esr.json [hunk 1]
```diff
@@ -23,31 +23,27 @@
     "shortcuts": [
         [
             "firefox.exe",
-            "Mozilla Firefox"
+            "Firefox Esr"
```
Comment (Ash258) [inline bucket/firefox-esr.json]: It is acronym.

```suggestion
            "Firefox ESR"
```

**Pull request merged**